src/generate_files.py

```diff
@@ -28,6 +28,9 @@
 - Expressive Synthetic Speech, [website](http://emosamples.syntheticspeech.de/)
 - Towards a standard set of acoustic features for the processing of emotion in speech, Technical university Munich, [document](https://asa.scitation.org/doi/pdf/10.1121/1.4739483)
 
+## Dataset loader
```

Comment:
Honestly, I am not sure about promoting another tool, especially if it is not a standard tool. 
The tool looks useful overall, but I am not sure if this is the right place or format to do it. 
Maybe instead of this, add at the end (after the disclaimer) a section called recommendation and use that to recommend the tool.

Reply:
OK, got it.

Reply:
great work :clap: